tool.sh

```diff
@@ -34,5 +34,5 @@ fi
 
 echo "If you see this, something went horribly wrong."
 echo "  1) Make sure you have crystal installed"
-echo "  2) Make sure `crystal` is in your \$PATH"
```

Comment:
Don't mix double- and single-quotes in the same "block".  The $PATH is already escaped to deal with that.

Reply:
it's not $PATH that will get expanded there, it's ``crystal``. We could do single quotes for the whole block?

Reply:
Ohhh good point.  Yes using single-quotes for the block would be fine!